src/schematics/src/update/update-0.13.0.ts

```diff
@@ -76,6 +107,55 @@ Start changes for Clarity 0.13.0
             );
           }
 
+          // Check for deprecated classes and update if possible
+          const reg = /import\s*{([^}]*)}\s*from\s*['"]@clr\/angular['"]/g;
+          let result = reg.exec(updated);
+          while (result !== null) {
+            const classRegex = /\b(?!Cl)\w+\b/g;
+            let importedClass = classRegex.exec(result[1]);
+
+            while (importedClass !== null) {
+              // Check for ones that were renamed
+              if (
+                importedClass[0].search(
+                  /Comparator|Filter|State|StringFilter|Checkbox|DatagridHideableColumnDirective|NavLevelDirective|TabLinkDirective|WizardPageButtonsDirective|WizardPageHeaderActionsDirective|WizardPageNavTitleDirective|WizardPageTitleDirective/g
```

Comment:
You can easily avoid this:
- In `getNewClassName`, make the default:
```typescript
default:
  return 'Clr'+oldClassname;
```
This makes it so that your function works for both hardcoded renames and generic prefixing.
- Remove this first `if()`, keep the other ones.
- In the final `else`, replace
```typescript
updated = updated.replace(re, 'Clr$&');
```
by
```typescript
updated = updated.replace(re, getNewClassname(importedClass[0]));
```